test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -438,7 +430,7 @@ TEST(cmod_utilityrate5_eqns, Test_Residential_TOU_Rates_net_billing_carryover_ap
 
     ssc_number_t cost_with_system;
     ssc_data_get_number(data, "elec_cost_with_system_year1", &cost_with_system);
-    EXPECT_NEAR(-115.6, cost_with_system, 0.1);
+    EXPECT_NEAR(-173.7, cost_with_system, 0.1);
 
```

Comment:
@brtietz, do you have a workbook to support these updates? Can you place in the SAM-documentation repo in the UtilityRates folder?

Reply:
I don't have a workbook, I was looking at the credit earned categories and making sure that the December values were the sum of the Nov credit and the Dec earned (or whatever is appropriate for another month). This SAM file duplicates the gen and load profiles from the unit tests and allows for more transparency: https://github.com/NREL/SAM-documentation/blob/master/UtilityRates/NetBilling/generic_system_ur_tests.sam Good enough?